config/locales/en.yml

```diff
@@ -62,6 +62,7 @@ en:
   certificates:
     caption: "%{heading}"
     header_id: ID
+    header_name: Name
```

Comment:
"Name" now exists 3 times in the yml file, we should be reusing it?

Reply:
I don't think it represents the same type of thing 3 times.